src/test/regress/expected/null_parameters.out

```diff
@@ -10,11 +10,12 @@ SELECT create_distributed_table('text_dist_column', 'key');
 
 (1 row)
 
-SET client_min_messages TO DEBUG;
+-- it seems that sometimes the pruning is deferred and sometimes not.
+-- what we care about is that these queries don't crash and the log for this
+-- one shouldn't matter. This is to prevent these test being from flaky in our CI.
+SET client_min_messages to ERROR;
```

Comment:
Prefer `NOTICE` to `ERROR` as while debugging tests I often sprinkle `WARNING` logging & it's confusing if the lack of effect is due to client_min_messages being `ERROR`